reactor-netty-core/src/main/java/reactor/netty/FutureMono.java

```diff
@@ -149,6 +157,8 @@ public void subscribe(CoreSubscriber<? super Void> s) {
 		final CoreSubscriber<? super Void> s;
 
 		final F                            future;
+
+		boolean cancelled;
 		FutureSubscription(F future, CoreSubscriber<? super Void> s) {
```

Comment:
nit: add empty row between field and constructor

Reply:
I have to check how to add this to checkstyle configuration ...

Reply:
done